client/src/utils/enums.ts

```diff
@@ -22,10 +22,10 @@ export const roleRestrictions: { id: string; name: string }[] = [
   { id: 'air', name: 'Air' },
   { id: 'earth', name: 'Earth' },
   { id: 'fire', name: 'Fire' },
-  { id: 'void', name: 'Void' },
```

Comment:
Here it's actually better to have the elements first (ordered alphabetically) and then have Keeper and Seeker.